.gitignore

```diff
@@ -0,0 +1,41 @@
+# Build Files
+**/CMakeFiles/
+**/CMakeCache.txt
+**cmake_install.cmake
+compile_commands.json
+Makefile
+**CTestTestfile.cmake
+
+# Visual Studio Code project files
+.vscode/
+language.settings.xml
+
+# Conan files
+conan.cmake
```

Comment:
You should not gitignore Conan.cmake. All the deps live here!!
```suggestion

```

Reply:
conan.cmake file is downloaded from the repo automatically the first time cmake runs. it's not part of the project.

Reply:
There is another file called Conan.cmake under cmake folder. This conflicts with that.

Reply:
thanks for pointing that out. I believe using '/conan.cmake' would fix this.